The_Lost_Star/common/activities/activity_types/expedition.txt

```diff
@@ -0,0 +1,1000 @@
+﻿activity_expedition = {
+	is_shown = {
+		highest_held_title_tier > tier_barony
+		is_landed = yes
+		is_ai = no
```

Comment:
Just an FYI, eventually we will want to remove is_ai = no
This is only for testing atm